backend/inspirehep/config.py

```diff
@@ -51,8 +51,8 @@
 INSPIRE_NEXT_URL = "http://localhost:5000"
 LEGACY_BASE_URL = "https://old.inspirehep.net"
 LEGACY_RECORD_URL_PATTERN = "http://inspirehep.net/record/{recid}"
-INSPIRE_BACKOFFICE_URL = "https://backoffice.dev.inspirebeta.net"
-AUTHENTICATION_TOKEN_BACKOFFICE = "CHANGE_ME"
+INSPIRE_BACKOFFICE_URL = "http://host.docker.internal:8001"
```

Comment:
so we dont modify it when we are doing tests every single time

Reply:
👍 We have to make sure it's set on k8s

Reply:
Added to the `hep-globals`
https://github.com/cern-sis/kubernetes-inspire/commit/b19919d806529302b4dd807de564f63512b05708